tests/README.md

```diff
@@ -0,0 +1,62 @@
+# Splunk OpenTelemetry Collector Integration Tests and Utilities
+
+To assist in vetting and validating the upstream and Splunk Collector distributions of the Collector, this library
+provides a set of integration tests and associated utilities.  The general testing pattern this project is geared toward
+is:
+
+1. Building the Collector (`make otelcol` or `make all`)
+1. Defining your expected [resource metric content](./testutils/README.md#resource-metrics) as a yaml file
+([see example](./testutils/testdata/resourceMetrics.yaml))
+1. Spin up your target resources as docker containers (TODO)
+1. Stand up an in-memory OTLP Receiver and metric sink capable of detecting if/when desired data are received (TODO).
+1. Spin up your Collector as a subprocess configured to report to this OTLP receiver (TODO)
```

Comment:
Does this PR do this?
Can we have a picture that shows what this PR does? I am a bit at a loss here, not sure I understand how things work.

Reply:
I've tried to express this in the PR description and denoting elements with `(TODO)` that will land in future PRs (like the current test containers one).